modules/auxiliary/server/ldap_server.rb

```diff
@@ -0,0 +1,100 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::LDAP::Server
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Native LDAP Server (Example)',
+        'Description' => %q{
+          This module provides a Rex based LDAP service to expose the
+          native Rex LDAP server functionality created during log4shell
+          development.
+        },
+        'Author' => [
+          'RageLtMan <rageltman[at]sempervictus>', # infrastructure
+          'Spencer McIntyre' # syntactically sane/correct Ruby LDAP object actions from early effort on l4j2 scanner
+        ],
+        'License' => MSF_LICENSE,
+        'References' => [],
+        'Actions' => [
+          [ 'Service', { 'Description' => 'Run LDAP server' } ]
+        ],
+        'PassiveActions' => [
+          'Service'
+        ],
+        'Notes' => {
```

Comment:
This needs the Default Action set in here otherwise you get this exception:
```
msf6 auxiliary(server/ldap_server) > run
[-] Auxiliary failed: Msf::MissingActionError Invalid action: Please use: Service
[-] Call stack:
```

```suggestion
        'DefaultAction' => 'Service',
        'Notes' => {
```